packages/edit-widgets/src/components/widget-areas-block-editor-provider/index.js

```diff
@@ -48,12 +49,14 @@ export default function WidgetAreasBlockEditorProvider( {
 			reusableBlocks: ALLOW_REUSABLE_BLOCKS
 				? select( coreStore ).getEntityRecords( 'postType', 'wp_block' )
 				: [],
-			isFixedToolbarActive: select(
-				editWidgetsStore
-			).__unstableIsFeatureActive( 'fixedToolbar' ),
-			keepCaretInsideBlock: select(
-				editWidgetsStore
-			).__unstableIsFeatureActive( 'keepCaretInsideBlock' ),
+			isFixedToolbarActive: select( interfaceStore ).isFeatureActive(
```

Comment:
Does this feature exist in the editor? I thought it was just in the customizer.

Reply:
It's in the standalone editor too:
<img width="272" alt="Screenshot 2021-08-11 at 12 51 52 pm" src="https://user-images.githubusercontent.com/677833/128971459-d30d2053-8fc4-4005-8a7a-ca3e2fa765eb.png">

Reply:
Sure, but it's not in the widgets editor, is it? I was wondering what it's doing in edit-widgets.

Reply:
Sorry, I don't quite understand. It is in the widgets editor. `edit-widgets` is the standalone widgets editor, and it has the fixed/top toolbar option.

If you're not seeing it in the menu, it might be because it's hidden below a certain breakpoint (https://github.com/WordPress/gutenberg/pull/32690).

Reply:
Oh right, so it does! I was looking at the bottom part of the menu where the other preferences are 🤦